client/src/components/PostedOn.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Typography } from "@mui/material";
+import { format } from "date-fns";
+import { useTheme } from "@emotion/react";
+
+//assuming the format of the props.date is YYYY-MM-DD
+const PostedOn = (props) => {
+  const postedDate = format(new Date(props.date), "M/d/yy");
+  let theme = useTheme();
```

Comment:
prefer const over let if not overriding

Reply:
We should ensure the prefer-const eslint rule is on